slack.js

```diff
@@ -105,3 +105,49 @@ export async function notifySlack(api, chainState, header, signedBlock) {
     }
   })
 }
+
+const MAX_CONSECUTIVE_FAILURES_BEFORE_MESSAGE = 5
+const CALL_INDEX = [64, 12]
+let failedConsecutiveSkycExtrinsics = 0
+
+export async function checkForSkycFailure(api, signedBlock) {
+  const skycDidTxs = signedBlock.extrinsics.filter(
+    ({ extrinsic }, index) => {
+      extrinsic.index = index
+      return extrinsic.isSigned && extrinsic.callIndex[0] === CALL_INDEX[0] && extrinsic.callIndex[1] === CALL_INDEX[1] && extrinsic.signer.toString() == process.env.SKYC_ADDRESS
+    }
+  )
+  console.log(`SKYC DID txs for block: [${skycDidTxs}]`)
```

Comment:
maybe don't log the txs only the number
```suggestion
  console.log(`SKYC DID txs for block: [${skycDidTxs.length}]`)
```